themes/classic/templates/_partials/form-fields.tpl

```diff
@@ -53,9 +53,10 @@
 
         {block name='form_field_item_country'}
           <select
-          class="form-control form-control-select js-country"
-          name="{$field.name}"
-          {if $field.required}required{/if}
+            id="{$field.name}"
+            class="form-control form-control-select js-country"
+            name="{$field.name}"
+            {if $field.required}required{/if}
           >
             <option value disabled selected>{l s='-- please choose --' d='Shop.Forms.Labels'}</option>
```

Comment:
Do you think the '-' will be read? like 'dash dash please choose dash dash'? 😅

Reply:
Maybe yes, I need to test it :p

Reply:
It's read by screen readers @Julievrz